package.json

```diff
@@ -62,20 +63,29 @@
     "view-helpers": "~0.1.3"
   },
   "devDependencies": {
+    "angular": "^1.6.4",
+    "angular-mocks": "^1.6.4",
+    "angular-ui-router": "^0.4.2",
```

Comment:
minor: I'm not sure why angular stuff is here, coz I see that in your karma config, you reference `public/lib/angular`, but angular being here means it's installed in the `node_modules` folder. These dependencies should probably be in `bower.json`.